.github/workflows/prod.yml

```diff
@@ -1,9 +1,12 @@
 name: Publish (prod)
 
 on:
-  push:
-    tags:
-      - v*.*.*
+  workflow_dispatch:
+    inputs:
+      ref:
```

Comment:
Isn't `ref` already available for `workflow_dispatch` events? Why do we need to specify this manually/duplicate the `ref`?

> When you trigger the event, you can provide the `ref` and any `inputs`

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

> 5. Select the Branch dropdown menu and click a branch to run the workflow on

https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow

Reply:
nice catch, didn't know this was an available built-in, i will incorporate it!

Reply:
Awesome! Thanks!